velox/exec/fuzzer/PrestoQueryRunner.cpp

```diff
@@ -384,6 +384,54 @@ std::optional<std::string> PrestoQueryRunner::toSql(
   return sql.str();
 }
 
+namespace {
+
+void appendWindowFrame(
```

Comment:
k Range Frames require some special handling here. For k Range frames, the computed range frame value is passed in the bound column. But for the Presto generated SQL we want the original (constant or column) bound. Presto will compute the range frame bound by itself. 

We will need to add some metadata to retrieve the original range bound from WindowNode for this.

Might be sufficient for you to just leave a comment here.

Reply:
I'll add a comment `// TODO: Add support for k Range Frames by retrieving the original range bound from WindowNode.`. Please let me know if this is not accurate. 

Also, do you have an example of the "k Range Frames"? Do you mean something like `RANGE BETWEEN c0 PRECEDING AND c1 FOLLOWING` and how is the computed range frame value different from the original bound?